app/models/application.rb

```diff
@@ -25,6 +25,13 @@ def latest_deploys_by_environment
       .compact
   end
 
+  def current_image_deployed_by_environment
```

Comment:
I'm struggling to follow what this method does. What is the logic checking `current_env == "production"` for? What do we display differently when viewing Release on Production vs Staging...?

Reply:
At the moment to limit potential security vulnerability the team decided that we should allow production to access the kubernetes API in any environment, but for integration and staging to only be able to access the kubernetes API in non-prod environments. This was in case we decide to change the allowable actions on the kubernetes API in the future.

Reply:
ok, I'll add something